stats/lib/handler.js

```diff
@@ -14,7 +14,9 @@ import {
   fetchRetrievalSuccessRate,
   fetchDealSummary,
   fetchDailyRetrievalResultCodes,
-  fetchDailyMinerRSRSummary
+  fetchDailyMinerRSRSummary,
+  fetchDailyRetrievalTimes,
```

Comment:
"Retrieval times" isn't self explanatory - it could for example also be the timestamps at which retrievals have been attempted. What about this?

```suggestion
  fetchDailyRetrievalTimings,
```

Reply:
See for example chrome dev tools also using "timings":

<img width="776" alt="Screenshot 2025-01-05 at 22 43 59" src="https://github.com/user-attachments/assets/60a941b4-af43-4ca0-b2d5-e452e214779f" />

Reply:
+1 to use `timings`

Reply:
Thanks for suggestion. I have renamed everything from `times` to `timings` including table, routes and functions.